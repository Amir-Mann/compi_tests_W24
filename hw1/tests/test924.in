Tb+BYYs4R[pnH1sK7Y 3.m$eJ%W.&+F 2bb5]pG'j|7T$%Di'pa>=YhPW80Vh k
,$_GQn