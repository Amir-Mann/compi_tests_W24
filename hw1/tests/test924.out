1 ID Tb
1 BINOP +
1 ID B
Error 
